### ISSUE TITLE
"Player" embed option tries to embed wrong url

### ISSUE THREAD

**Issue by jbenet** [asciinema/asciinema.org#194]
**TL;DR: in the "player" embed code, the attribute `id="asciicast-23535"` is incorrect. it should be `id="asciicast-<id-of-cast>"`. example:

```html
<script type="text/javascript" src="https://asciinema.org/a/8pgmezlut3880svafm3880vzl.js" id="asciicast-23535" async></script>
```

---

Long version:

the js embedded with the "player" embed option:

![](http://gateway.ipfs.io/ipfs/QmUoowwU6K74twf4eLtQ5YgkFRfXTDVzqekcDM9KuZsib9/cap.png)

quick test page

```html
<html><head></head>
<body>
  <script type="text/javascript" src="https://asciinema.org/a/8pgmezlut3880svafm3880vzl.js" id="asciicast-23535" async></script>
</body>
</html>
```

which, when run, turns into:

```html
<html><head>
</head>
<body>

  <script type="text/javascript" src="https://asciinema.org/a/8pgmezlut3880svafm3880vzl.js" id="asciicast-23535" async="" data-player="[object HTMLDivElement]"></script><div id="asciicast-container-23535" class="asciicast" style="display: block; float: none; overflow: hidden; padding: 0px; margin: 20px 0px;"><iframe src="https://asciinema.org/api/asciicasts/23535" id="asciicast-iframe-23535" name="asciicast-iframe-23535" scrolling="no" allowfullscreen="true" style="overflow: hidden; margin: 0px; border: 0px; display: inline-block; width: 100%; float: none; visibility: visible;"></iframe></div>
</body></html>
```

the `iframe` src is https://asciinema.org/api/asciicasts/23535 which shows me: 

![](http://gateway.ipfs.io/ipfs/QmVPscBi5VCEzZiRFHKisQojT9zYS94UkNqYoJTtLV956r/cap.png)

this is because the code `23535` is wrong. it should be https://asciinema.org/api/asciicasts/8pgmezlut3880svafm3880vzl

**Comment by sickill** [comment]
Looks like a bug indeed, It's a problem with private asciicasts embedding. It's a quick fix.
